### DISCUSSION TITLE
Links to events shouldn't say (comment)

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#63487] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

- Related to https://github.com/community/community/discussions/12009

Consider this comment, https://github.com/github/docs/pull/25401#issuecomment-1674710307 which when clicked will yield:
<img width="1154" alt="image" src="https://github.com/community/community/assets/2119212/ce557edd-9918-4c59-9e9b-a14a499119af">

It says:
> `@cmwilson21` https://github.com/github/docs/pull/25401#event-10067498164 still seems unhappy? 😦

Depending on your entrypoint to the PR, at best, you'll see:

<img width="1146" alt="image" src="https://github.com/community/community/assets/2119212/c7130a82-9990-47f6-892a-92b0c87dc338">

Note that the link said it was a `(comment)`, but it isn't a comment, the url clearly indicates it's an `event` and there is no comment text, not from me, and not from `GitHub Actions`.

The current behavior is misleading and confusing.

**Comment by EdamAme-x** [comment]
If you change your perspective, you can say that it's also a comment,
But, it would be better to specify it as an event at least. 
I agree with your opinion.

**Reply by jsoref** [reply]
Here's my opinion based on the official api:
https://api.github.com/repos/github/docs/issues/25401/comments ... does not list these things, and therefore they are *not* comments.

**Reply by EdamAme-x** [reply]
If the offical api say it, it can't be helped.

This should be specified as an event.
